### ISSUE TITLE
cannot download bed files for dm6 genome

### ISSUE THREAD

**Issue by jdmontenegro** [inutano/chip-atlas#87]
Hi, 
I am trying to download embryonic ATACseq bedgraph data from the dm6 genome. I keep getting an error as resource unavailable. 
Could you point me to the right direction where I could get this data?

Kind regards,

Juan D. Montenegro

**Comment by shinyaoki** [comment]
Hi Juan,

Thank you for using ChIP-Atlas. Could you tell me the accession ID (SRX*****)?

Shinya

**Comment by shinyaoki** [comment]
Hi Juan,

We have looked into the problem and found that most of the dm6 data on Peak Browaser has been lost by unknown reason. We will fix the data by the time of the next update. Currently, dm3 data is entirely available, so please use the data of dm3 assembly.

We sincerely apologize for the inconvenience.

Shinya

**Comment by jdmontenegro** [comment]
Hi Shinya,

Thank you for your reply. On a related note, I downloaded the dm3 atacseq
data and did the basic analysis of peak distribution along the dm3 genome
(biocoductor). I found that dm3 has most of the peaks very fat away from
annotated genes (around 70% of peaks fall very far away from genes). This
is a clear outlier compared to the other species and I think it might be
some problem either with the annotation or the peak calls. Perhaps a
different genome version was used instead?
I can share those results later today.

Kind regards,

Juan D. Montenegro
